_ml-commons-plugin/algorithms.md

```diff
@@ -629,4 +629,79 @@ POST _plugins/_ml/_predict/logistic_regression/SsfQaoIBEoC4g4joZiyD
 
 ### Limitations
 
-Convergence metrics are not built into Tribuo's trainers. Therefore, ML Commons cannot indicate the convergence status through the ML Commons API.
\ No newline at end of file
+Convergence metrics are not built into Tribuo's trainers. Therefore, ML Commons cannot indicate the convergence status through the ML Commons API.
+
+## Metrics correlation
+
+The metrics correlation feature is an experimental feature released in OpenSearch 2.7. It can't be used in a production environment. To leave feedback on improving the feature, leave an issue in the [ML Commons repository](https://github.com/opensearch-project/ml-commons).
```

Comment:
Overall looks good to me. `It can't be used in a production environment.` seems bit unnecessary. Generally experimental feature means not ready for prod environment